src/server/logging/applyFilterToKey.js

```diff
@@ -0,0 +1,23 @@
+function replacer(match, group) {
+  return (new Array(group.length + 1).join('X'));
+}
+
+module.exports = function applyFilterToKey(obj, key, action) {
+  for (let k in obj)  {
+    if (obj.hasOwnProperty(k)) {
+      let val = obj[k];
+      if (k === key) {
+        val = ''  + val;
+        if (action === 'remove') delete obj[k];
+        if (action === 'censor') {
```

Comment:
See https://github.com/elastic/kibana/pull/5038/files#r40466593, maybe we should always use `remove` when `val` is an object, or censor all of it's properties recursively?